XML/Other/ReplaceableTestParameters.xml

```diff
@@ -274,7 +274,7 @@ Scenario 2 : You need to run all the performance tests quickly.
 	</Parameter>
 	<Parameter>
 		<ReplaceThis>INFINIBAND_INTEL_NBC_TESTNAMES</ReplaceThis>
-		<ReplaceWith>allreduce</ReplaceWith>
+		<ReplaceWith></ReplaceWith>
```

Comment:
why remove this?

Reply:
This parameter deprecated in Intel MPI 2019 Update 6